javascript/javascript2/week3/Fastpresser/index.js

```diff
@@ -0,0 +1,127 @@
+const button = document.getElementById("start-button");
+const p = document.getElementById("count-down");
+const left = document.getElementById("left");
+const right = document.getElementById("right");
+const input = document.getElementById("input");
+const leftPlayer = document.querySelector(".leftPlayer");
+const rightPlayer = document.querySelector(".rightPlayer");
+
+var confettiElement = document.getElementById("my-canvas");
+var confettiSettings = { target: confettiElement };
+var confetti = new ConfettiGenerator(confettiSettings);
```

Comment:
use const instead of var ;)

Reply:
Yes ... I have to pay more attention ... this code was from the git homework. But yes I have to change it when I use it.